### PR TITLE
copy updates per marketing

### PR DESCRIPTION
**Pull request by Dhaulagiri**
### :pushpin: Summary

Marketing requested some light updates to the verbiage on the home page and the about page

https://hds-website-27wczufki-hashicorp.vercel.app/

https://hds-website-27wczufki-hashicorp.vercel.app/about

### :link: External links

Jira ticket: [HDS-1451](https://hashicorp.atlassian.net/browse/HDS-1451)

[HDS-1451]: https://hashicorp.atlassian.net/browse/HDS-1451?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: website/app/templates/index.hbs [hunk 1]
```diff
@@ -2,8 +2,8 @@
   <div class="doc-page-home__content">
     <div class="doc-page-home__hero">
       <h1><span class="doc-page-home__hero-helios">Helios</span> Design System</h1>
-      <p>The Helios Design System provides the building blocks to design and implement consistent, accessible, and
-        delightful product experiences across HashiCorp.</p>
+      <p>The Helios Design System from HashiCorp provides the open source building blocks to design and implement
+        consistent, accessible, and delightful product experiences</p>
```
Comment (MelSumner) [inline website/app/templates/index.hbs:6]:
```suggestion
        consistent, accessible, and delightful product experiences.</p>
```

File: website/config/environment.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ module.exports = function (environment) {
 
     'ember-meta': {
       description:
-        'Helios Design System provides the building blocks to design and implement consistent, accessible, and delightful product experiences across HashiCorp.',
+        'The Helios Design System from HashiCorp provides the open source building blocks to design and implement consistent, accessible, and delightful product experiences',
```
Comment (MelSumner) [inline website/config/environment.js:33]:
```suggestion
        'The Helios Design System from HashiCorp provides the open-source building blocks to design and implement consistent, accessible, and delightful product experiences.',
```

I believe "open source" is used here as an adjective and as such should be hyphenated.

**Review by MelSumner — CHANGES_REQUESTED** [review]

File: website/app/templates/about.hbs [hunk 3]
```diff
@@ -20,8 +20,8 @@
     </p>
 
     <h2 class="doc-text-h2">Contributions</h2>
-    <p class="doc-text-body">As an open-source project, we think a lot about how people can contribute—and there are
-      quite a few ways our consumers are encouraged to participate! Read our
+    <p class="doc-text-body">As an open source project, we think a lot about how people can contribute—and there are
+      quite a few ways our consumers are encouraged to participate. Read our
       <a class="doc-link-generic" href="/about/contribution/" target="_blank" rel="noopener noreferrer">contribution</a>
       documentation to find the different ways anyone can contribute to Helios.
```
Comment (KristinLBradley) [inline website/app/templates/about.hbs:26]:
```suggestion
      documentation to find the different ways someone can contribute to Helios.
```

Comment (KristinLBradley) [reply website/app/templates/about.hbs:26]: Either is ok but I'm not sure we should imply just "anyone" can contribute.

**Review by KristinLBradley — APPROVED** [review]
Use of the term "open source" in here still seems potentially confusing to me...

**Comment by Dhaulagiri** [comment]
I moved this back to draft as I'm still working on this

File: website/app/templates/about.hbs (hunk 3 above)
Comment (KristinLBradley) [reply website/app/templates/about.hbs:26]: Could also change the "can" to "could" in here to further soften a bit the implication that others can contribute in the way normally understood in "open source" projects.

File: website/config/environment.js (hunk 2 above)
Comment (Dhaulagiri) [reply website/config/environment.js:33]: see my message in slack but we have decided as a company not to hyphenate

File: website/app/templates/about.hbs (hunk 3 above)
Comment (Dhaulagiri) [reply website/app/templates/about.hbs]: added these in, thanks!

**Comment by Dhaulagiri** [comment]
Ok, I re-confirmed this update with Marketing. I've remove some of the open source references and adjusted the contribution section to be more explicit that we are talking about contributions from HashiCorp consumers

File: website/app/templates/about.hbs [hunk 4]
```diff
@@ -3,11 +3,11 @@
 <Doc::Page::Stage>
   <Doc::Page::Cover @title="About" />
   <Doc::Page::Content>
-    <p class="doc-text-body">Helios is a flexible, scalable design system built on the foundations of accessibility,
-      beautiful design, and consistent user experience.</p>
+    <p class="doc-text-body">Helios is a flexible, scalable design system built by HashiCorp on the foundations of
+      accessibility, beautiful design, and consistent user experience..</p>
```
Comment (Dhaulagiri) [inline website/app/templates/about.hbs:7]:
```suggestion
      accessibility, beautiful design, and consistent user experience.</p>
```

File: website/app/templates/about.hbs [hunk 5]
```diff
@@ -20,10 +20,11 @@
     </p>
 
     <h2 class="doc-text-h2">Contributions</h2>
-    <p class="doc-text-body">As an open-source project, we think a lot about how people can contribute—and there are
-      quite a few ways our consumers are encouraged to participate! Read our
+    <p class="doc-text-body">As an open source project, we think a lot about how people can contribute—and there are
+      quite a few ways our consumers at HashiCorp are encouraged to participate. Read our contribution documentation to
+      find the different ways anyone can contribute to Helios.. Read our
```
Comment (Dhaulagiri) [inline website/app/templates/about.hbs:25]:
```suggestion
      find the different ways anyone can contribute to Helios. Read our
```

File: website/config/environment.js (hunk 2 above)
Comment (Dhaulagiri) [inline website/config/environment.js:33]:
```suggestion
        'The Helios Design System from HashiCorp provides the open source building blocks to design and implement consistent, accessible, and delightful product experiences.',
```

File: website/app/templates/about.hbs [hunk 6]
```diff
@@ -20,10 +20,11 @@
     </p>
 
     <h2 class="doc-text-h2">Contributions</h2>
-    <p class="doc-text-body">As an open-source project, we think a lot about how people can contribute—and there are
-      quite a few ways our consumers are encouraged to participate! Read our
+    <p class="doc-text-body">As an open source project, we think a lot about how people can contribute—and there are
+      quite a few ways our consumers at HashiCorp are encouraged to participate. Read our contribution documentation to
+      find the different ways anyone can contribute to Helios. Read our
```
Comment (heatherlarsen) [inline website/app/templates/about.hbs:25]:
```suggestion
      quite a few ways our consumers at HashiCorp are encouraged to participate. Read our
```

**Review by heatherlarsen — APPROVED** [review]
One minor change

**Review by didoo — APPROVED** [review]
👍

**Review by MelSumner — APPROVED** [review]
:shipit:

**Pull request merged**